### DISCUSSION TITLE
🌟 🎁  FREE Certifications Voucher: Nominate an Outstanding Community Member  :octocat:

### DISCUSSION THREAD

**Discussion by queenofcorgis** [community/community#114271] [Announcements]
Our community was designed for our users to help one another. The power of multiple minds cannot be understated and it has been amazing to witness as our community grows.

It is especially awesome to see some of our community members be standout helpers across categories and we're grateful for their time, expertise, and kindness. However, with over 6000 comments created in the community each month, we want to make sure we don't miss a great community citizen. Lucky for us, there are no better experts on who has been doing an outstanding job than our community members.

## How do I nominate someone? 
⭐  Share their handle below in the comments and the reason why you're nominating them by **10am EST on Monday, April 15th.** 

Need an example of what makes a good community member - review our check-ins for some ideas:
- [Actions Check-in](https://github.com/orgs/community/discussions/103166)
- [API and Webhooks Check-in](https://github.com/orgs/community/discussions/112483)
- [Copilot Check-in](https://github.com/orgs/community/discussions/113541) 
- [Education Check-in](https://github.com/orgs/community/discussions/109690)
- [New to GitHub Check-in](https://github.com/orgs/community/discussions/112479)

## Why nominate someone? 
We may have eyes and ears everywhere in the community, but your perspective is just as important and we need your help to know who has been a great community helper. A little recognition goes a long way and they can benefit from getting GitHub certified. 

## What else should I know? 
Members cannot nominate themselves for this and nominations must be in by **10am EST on April 15, 2024.** 

Further details about the voucher and how it will be applied will be privately messaged to those selected to receive it. 

Please note all nominations are subject to review by the GitHub Community team and nominating a member does not come with any guarantees. Those coordinating nominations or fraudulent activity may be subject to our temporary or indefinite blocks. See our [Code of Conduct](https://github.com/community/community/blob/62ee2a8c8e5c8eba3a40ee41252418782764effd/CODE_OF_CONDUCT.md) for more.

**Comment by AR-AbdulAhad** [comment]
I would like to nominate [@ar-abdulahad] for the GitHub Certification exam voucher. They have consistently shown dedication, expertise, and kindness in helping fellow community members across various categories. Their contributions have greatly benefited our community, and I believe they deserve this recognition and opportunity. Thank you for considering my nomination!

**Reply by AlFer2** [reply]
Muito bom faça isso manoh

**Comment by digital180** [comment]
Gg

**Comment by rayhanall** [comment]
![Uploading playground-image-IYYU1.png…]()

**Comment by DeathKnight813** [comment]
Good

**Comment by Parasos** [comment]
It's all good thanks again.. but??

**Comment by gracetyy** [comment]
I would like to nominate @queenofcorgis for maintaining GitHub Community well

**Comment by belumume** [comment]
Since I'm not active here and I can't nominate myself, I nominate @queenofcorgis (she's the Most helpful according to the community dashboard ranking) even though she might be a Github employee and not a verified student.

**Comment by antresssss** [comment]
I nominate @queenofcorgis , a very cool username . Corgis! yes .

**Comment by 0YJ** [comment]
cool cool cool

**Comment by SunderSingh27** [comment]
amazing. I am active and dedication

**Comment by PedroPaino** [comment]
thank you, this is very cool and new for me😎

**Comment by ravindrasinghh** [comment]
Great Initiative,  I am actively and dedicatedly contributing to GitHub.! 
[https://github.com/ravindrasinghh]
![Uploading Screenshot 2024-04-16 at 8.59.03 AM.png…]()

**Comment by marxwahlorian** [comment]
> Our community was designed for our users to help one another. The power of multiple minds cannot be understated and it has been amazing to witness as our community grows.
> 
> It is especially awesome to see some of our community members be standout helpers across categories and we're grateful for their time, expertise, and kindness. However, with over 6000 comments created in the community each month, we want to make sure we don't miss a great community citizen. Lucky for us, there are no better experts on who has been doing an outstanding job than our community members.
> 
> ## Nominate a fellow community member who you think is doing an awesome job contributing by 10am EST on April 15, 2024.
> ### We will review all nominations and select a small group of winners to receive a voucher to take a [GitHub Certification exam](https://docs.github.com/en/get-started/showcase-your-expertise-with-github-certifications/about-github-certifications) for free!
> ## How do I nominate someone?
> ⭐ Share their handle below in the comments and the reason why you're nominating them by **10am EST on Monday, April 15th.**
> 
> Need an example of what makes a good community member - review our check-ins for some ideas:
> 
> * [Actions Check-in](https://github.com/orgs/community/discussions/103166)
> * [API and Webhooks Check-in](https://github.com/orgs/community/discussions/112483)
> * [Copilot Check-in](https://github.com/orgs/community/discussions/113541)
> * [Education Check-in](https://github.com/orgs/community/discussions/109690)
> * [New to GitHub Check-in](https://github.com/orgs/community/discussions/112479)
> 
> ## Why nominate someone?
> We may have eyes and ears everywhere in the community, but your perspective is just as important and we need your help to know who has been a great community helper. A little recognition goes a long way and they can benefit from getting GitHub certified.
> 
> ## What else should I know?
> Members cannot nominate themselves for this and nominations must be in by **10am EST on April 15, 2024.**
> 
> Further details about the voucher and how it will be applied will be privately messaged to those selected to receive it.
> 
> Please note all nominations are subject to review by the GitHub Community team and nominating a member does not come with any guarantees. Those coordinating nominations or fraudulent activity may be subject to our temporary or indefinite blocks. See our [Code of Conduct](https://github.com/community/community/blob/62ee2a8c8e5c8eba3a40ee41252418782764effd/CODE_OF_CONDUCT.md) for more.

Hello World , Anyone want a boost or are you ok? 
![Uploading avatar-1001548202574.jpg…]()

**Comment by marxwahlorian** [comment]
Wiseone
[Uploading BjÃ¶rn_WistrÃ¶m_-_ (4) (1).pdf…]()

**Comment by marxwahlorian** [comment]
#@_[
<details><summary>Details</summary>
<p>

> __

</p>
</details> ](url)_

**Comment by usamaabutt** [comment]
Interesting

**Comment by Krzysztof346** [comment]
Najlepszy Git na świecie

**Reply by Innu38** [reply]
Ok

**Comment by sudoyasir** [comment]
I'd like to nominate for @queenofcorgis 😄

**Comment by Nicroe69** [comment]
I’m just learning how to navigate the site and truth be told, trying to figure out how I access it lol. But I do believe it is awesome and very much needed to promote inclusive growth and trust throughout the world. Very interested in the building and production of everything and I will nominate https://github.com/queenofcorgis for the inspiration to stick to the tasks that are called for us and promoting rewards for our efforts! Very nice!

**Reply by Fundz555** [reply]
Ok

**Reply by abdimk** [reply]
⭐⭐⭐⭐⭐⭐⭐⭐⭐

**Comment by J13PhantomByte** [comment]
awesome #1

**Reply by Fundz555** [reply]
Hey

**Comment by Gergo146** [comment]
@queenofcorgisa kiemelt

**Comment by abdimk** [comment]
NICE  :)

**Comment by Mdshahzeley** [comment]
md join u wll platfon  coin power kod  from malaysia  ... 368ed u mlkjwmhi2024  dgi mm04 09ujbiyf  tekit  dll

**Comment by lucien-d** [comment]
good
